koap/build.gradle.kts

```
@@ -53,3 +53,43 @@ npmPublishing {
         }
     }
 }
+
+tasks.register("apiTestNpmInstall") {
+
+    description = "Builds the JS Koap package and installs it to the test suite for use"
+    group = "Verification"
+    dependsOn("assembleJsNpmPublication")
+
+    doLast {
+        val response = runCommand("cd apiTests && npm install --force file://../build/publications/npm/js")
+        println("$response")
+    }
+}
```

We should probably switch this (and the tasks below) to [`Exec` based tasks](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Exec.html).

Then you could drop the `cd` portion of all the commands and just define the working directory.

I _think_ something like the following would work...?

```suggestion
task<Exec>("apiTestNpmInstall") {
    description = "Builds the JS Koap package and installs it to the test suite for use"
    group = "Verification"
    dependsOn("assembleJsNpmPublication")
    workingDir("apiTests")
    commandLine("npm", "install", "--force", "file://../build/publications/npm/js")
}
```

---

runCommand() is not being used now. Want me to get rid of it?

---

> runCommand() is not being used now. Want me to get rid of it?

Yes, please. 😄